### DISCUSSION TITLE
Tilting the scroll wheel left and right to switch workspaces without needing to hover the cursor on the i3 bar.

### DISCUSSION THREAD

**Discussion by josh5555s** [regolith-linux/regolith-desktop#830] [Q&A]
When hovering on the i3 bar, tilting the scroll wheel left and right switches workspaces. How can I configure Regolith so that the same workspace switching behavior occurs no matter where the mouse cursor is located on the screen?

I've been considering the use of an application that handles input devices, like: [https://github.com/sezanzeb/input-remapper](https://github.com/sezanzeb/input-remapper), but if possible I would rather achieve the functionality within the Regolith configs. 

Much thanks for any help.

**Comment by harold** [comment]
This is an interesting question, what did you come up with @josh5555s ?

@kgilmer - I have a similar question. How can I configure regolith to _not_ change workspaces when mousewheel-ing on the i3 bar?
 - I find myself accidentally switching workspaces when scrolling near the bottom of webpages for instance.


Thanks all! Still living inside regolith every day and loving it! :bow:
